Rakefile

```diff
@@ -446,14 +450,13 @@ namespace :jsondoc do
       Rake::Task["jsondoc:google_cloud"].invoke(google_cloud_version, gh_pages_dir)
     end
 
-    # TODO: Uncomment following lines when ready to release stackdriver gem
-    # stackdriver_gems = ["stackdriver", "google-cloud-logging", "google-cloud-error_reporting", "google-cloud-monitoring"]
-    # if stackdriver_gems.include? gem
-    #   stackdriver_version = manifest_versions["stackdriver"]
-    #   header "Assembling jsondoc for stackdriver package"
-    #   puts "Latest stackdriver package version is '#{stackdriver_version}' (from docs/manifest.json)"
-    #   Rake::Task["jsondoc:stackdriver"].invoke(stackdriver_version, gh_pages_dir)
-    # end
+    stackdriver_gems = ["stackdriver", "google-cloud-logging", "google-cloud-error_reporting", "google-cloud-monitoring"]
```

Comment:
I think since the stackdriver gem doesn't directly depend on the google-cloud-core gem, the stackdriver gem jsondoc shouldn't depend on google-cloud-core in the first place. So when we package google-cloud-core gem, we don't need to update stackdriver jsondocs.